src/idle.c

```diff
@@ -73,8 +74,9 @@ int idle_requested(struct fmt_main *format)
 
 void idle_init(struct fmt_main *format)
 {
-#if !(defined(__MINGW32__) || defined (_MSC_VER) || defined(__BEOS__) || defined(__HAIKU__))
-	static int once;
+#if !defined(__BEOS__) && !defined(__HAIKU__) &&	  \
+	!defined(__MINGW32__) && !defined (_MSC_VER)
+	int old_nice;
 #endif
```

Comment:
BTW, we'd simplify this logic a lot by allowing mixed declarations and code. I no longer recall, why is it we don't allow those? I guess I was the one to support such restrictions some years back, but perhaps we no longer support a platform where they'd be relevant?

Reply:
I can't recall who wanted that or why. I kinda like it but we don't need to enforce it, especially in cases like this